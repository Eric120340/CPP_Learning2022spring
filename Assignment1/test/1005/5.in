4 4 4386051
610446003 10819871 973870786 207662558 
143512851 209222431 938140794 529468601 
487530415 130922941 525057024 79299563 
979399355 911257220 609210072 544595211 
