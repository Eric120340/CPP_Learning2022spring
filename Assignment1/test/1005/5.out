260677687 893691021 328160037 906788090 
484826694 681795293 811597451 173188768 
116195179 550859957 183778738 110352072 
353845041 998587269 341566667 733860558 
